lib/galaxy/objectstore/irods.py

```diff
@@ -350,27 +350,16 @@ def _download(self, rel_path):
         data_obj = None
 
         try:
-            data_obj = self.session.data_objects.get(data_object_path)
+            cache_path = self._get_cache_path(rel_path)
```

Comment:
Must pass the path to get(), so get() can download the file utilizing multiple threads, if necessary. Getting data object instance, reading and writing it to cache will not utilize multithreading and is removed.